en/flight_controller/pixhack_v3.md

```diff
@@ -0,0 +1,52 @@
+# Pixhack V3
+
+The CUAV *Pixhack V3* autopilot is a flexible autopilot intended primarily for manufacturers of commercial systems.  
```

Comment:
@cuavbin Is this whole section correct? I ask because it looks a lot like the information that applies to https://docs.px4.io/en/flight_controller/pixhawk-2.html and it does not match the information you have in your documentation http://doc.cuav.net/PixHack/pixhack-v3.html.

- Can you confirm that this is based on **FMUv3** . If not, what FMU version is it?
- Can you confirm that this is based on the PH2? If not, what design is it closest to?
- Can you confirm that this has a separate carrier board. If so, then can you show an image of that?

If not, then this should have a quick summary of the key selling point of your boards. It could say things like
- Fast processor with failsafe processor
- Redundancy design
- Lots of expansion
- Built-in shock absorption system

Thanks!

Reply:
Hello, hamishwillee
Pixhack v3 really is based on PH2 (SOLO) hardware to develop
One of China's most popular flight controllers.
We have done a lot of optimization, such as damping system and thermostat system, as well as the interface layout optimization.
The documentation may be Chinese at this time, and we are about to switch it automatically according to the viewer's language. 
We also open the design files at our cuav GitHub
https://github.com/cuav/CUAV_Hardware/tree/master/Pixhack_V3

Reply:
Thanks. So are there any carrier boards?